ietf-microwave-radio-link.yang

```diff
@@ -276,16 +289,31 @@ module ietf-microwave-radio-link {
       "Addition of data nodes for carrier termination to
        the standard Interface data model, for interfaces
        of the type 'microwaveCarrierTermination'.";
+   leaf carrier-id-enabled {
+      type boolean;
+      description
+        "'true' value when the check if far-end CT is connected 
+          to the correct near-end CT is used.
+         'false' if the check is not used.";
+    }
     leaf carrier-id {
       type string;
       default "A";
       description
-        "ID of the carrier (e.g., A, B, C, or D).
-         Used in XPIC and MIMO configurations to check that
-         the carrier termination is connected to the correct
-         far-end carrier termination.  Should be the same
-         carrier ID on both sides of the hop.  Left as
-         default value when MIMO and XPIC are not in use.";
+        "ID of the near-end carrier (e.g., A, B, C, or D).
+         Used in XPIC and MIMO configurations (or in RLT configurations 
+         with more one CT) to check that the carrier termination is 
+         connected to the correct far-end carrier termination. 
+         Left as default value when not used.";
```

Comment:
I have the same doubt as above about the case where "not used".

Moreover, does this mean that, when used, the default value A should not be used?